### ISSUE TITLE
Sympa Accessibility Issue - 8  - Link text must describe the link target

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1766]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
Link text must describe the link target.  The accessible name of a link must accurately describe the target or purpose of the link.

Actual behavior
---------------
See image below.

Link text are not describing the link target for the Screen Reader to read.

Steps to reproduce
------------------
Sympa Accessibility Issue - 8 - was determined by an Automation Tool Evaluation using AInspector and Siteimprove Accessibility Checkers as automated accessibility evaluation tools.

After running these automation tools the following issue was identified:

Link text must describe the link target. The accessible name of a link must accurately describe the target or purpose of the link. When the accessible name of a link does not describe its target or purpose, users will not have the information they need to determine the usefulness of the target resources.

[WCAG Level A 2.4.4 Link Purpose (In Context) (Primary)](https://www.w3.org/TR/WCAG21/#link-purpose-in-context)

![image](https://github.com/sympa-community/sympa/assets/144282683/63ce449d-8280-4298-8b6d-eee893578d04)

Fig 8 Link text without link target.

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13549692/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)

**Comment by ikedas** [comment]
Hi @rm1-uiuc ,

All of the elements you have pointed out have a `title` attribute describing the link targe, but is this insufficient? If it is, I would appreciate your advice on what measures could be taken.
